### ISSUE TITLE
Error after upgrading to nest 6 and ng-universal 0.4.0

### ISSUE THREAD

**Issue by tuurbo** [nestjs/ng-universal#21]
## I'm submitting a...

<pre><code>
[ ] Regression 
[x] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

Updated to nest 6 and ng-universal 0.4.0
```
"@nestjs/common": "^6.0.1",
"@nestjs/core": "^6.0.1",
"@nestjs/ng-universal": "^0.4.0",
"@nestjs/testing": "^6.0.1",
"@nestjs/platform-express": "^6.0.1",
```

Getting the error below...

###############
[Nest] 7920   - 03/18/2019, 3:58 PM   [NestFactory] Starting Nest application...
[Nest] 7920   - 03/18/2019, 3:58 PM   [ExceptionHandler] Nest can't resolve dependencies of the AngularUniversalModule (ANGULAR_UNIVERSAL_OPTIONS, ?). Please make sure that the argument at index [1] is available in the AngularUniversalModule context. +22ms
###############

```ts
// main.ts
import { enableProdMode } from '@angular/core';
import { NestFactory } from '@nestjs/core';
import { NestExpressApplication } from '@nestjs/platform-express';
import { ApplicationModule } from './app.module';

enableProdMode();

async function bootstrap() {
    const app = await NestFactory.create<NestExpressApplication>(ApplicationModule);

    app.enableCors({
        methods: 'GET',
        maxAge: 3600,
    });

    await app.listen(process.env.PORT);
}

bootstrap().catch(err => console.error(err));

```
```ts
// app.module.ts
import { Module } from '@nestjs/common';
import { AngularUniversalModule } from '@nestjs/ng-universal';
import { join } from 'path';

import { HelmetModule } from './middlewares/helmet.module';
import { SessionModule } from './middlewares/session.module';
import { RoutesController } from './routes/routes.controller';

@Module({
    imports: [
        HelmetModule,
        SessionModule,
        AngularUniversalModule.forRoot({
            viewsPath: join(process.cwd(), 'dist/app'),
            bundle: require('./../dist/server/main.js'),
        }),
    ],
    controllers: [RoutesController],
})
export class ApplicationModule { }
```

Let me know if you need anything else. Thanks

**Comment by kamilmysliwiec** [comment]
Fixed in 4.2.0 :)

**Comment by tuurbo** [comment]
Thanks :)
